Cargo.toml

```diff
@@ -12,6 +12,7 @@ homepage = "https://www.parity.io/"
 description = "Encode and decode values of arbitrary shapes to SCALE bytes"
 keywords = ["parity", "scale", "encoding", "decoding"]
 include = ["Cargo.toml", "src/**/*.rs", "README.md", "LICENSE"]
+rust-version = "1.81.0"
 
 [features]
 default = ["std", "serde", "from-string", "parser-ss58"]
```

Comment:
I reckon we can remove the `std` feature from this when:

- We relegate `scale-info` to be a dev dependency: we don't use it in the main lib, only for tests and doc (we expose `PortableRegistry in a couple of places but it's not actually needed anywhere). Then we don't need to care about std on it or whatever.
- The PRs for scale-encode/scale-decode (/scale-bits too?) are released. I don't think the either/serde/serde_json std things need to be toggled or whatever so they prob don't matter.